penrose-web/src/engine/Evaluator.ts

```diff
@@ -221,6 +213,21 @@ function toFloatVal<T>(a: ArgVal<T>): T {
   }
 }
 
+function toVecVal<T>(a: ArgVal<T>): T[] {
+  if (a.tag === "Val") {
+    const res = a.contents;
+    if (res.tag === "VectorV") {
+      return res.contents;
+    } else {
+      console.log("res", res);
```

Comment:
Will need to clean up these logs soon

Reply:
Yeah, I only left in `console.log` right before an error is thrown, for debugging.